android/filament-utils-android/CMakeLists.txt

```diff
@@ -16,6 +16,16 @@ include_directories(${FILAMENT_DIR}/include
         ..
         ../../libs/utils/include)
 
+set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -fno-stack-protector")
```

Comment:
This reminds me we should really switch all of our CMake files to the proper way of setting compiler flags. This is the old hacky way.

Reply:
Agreed, we should do a clean up pass at some point.